GlobalExtensions.php

```diff
@@ -33,6 +33,7 @@
 	'OATHAuth',
 	'OAuth',
 	'ParserFunctions',
+	'QuickInstantCommons',
```

Comment:
Is it global?

It only needs to be on certain wikis albeit most.

Reply:
It is a config that disables main functionality, in order to use for miraheze Commons, and shared uploads, the extension itself must be global.